src/docs/torque.md

```diff
@@ -503,10 +503,29 @@ macro Bar(implicit context: Context)() {
 ```
 
 In contrast to Scala, we forbid this if the names of the implicit parameters are not identical.
-For overload resolution not to cause fragile and confusing behavior, we want that the implicit parameters do not influence overload resolution at all. That is: when comparing candidates of an overload set, we do not consider the available implicit bindings at the call-site. Only after we found a single best overload, we check if implicit bindings for the implicit parameters are available.
+
+Since overload resolution can cause confusing behavior, we ensure that implicit parameters do not influence overload resolution at all. That is: when comparing candidates of an overload set, we do not consider the available implicit bindings at the call-site. Only after we found a single best overload, we check if implicit bindings for the implicit parameters are available.
 
 Having the implicit parameters left of the explicit parameters is different from Scala, but maps better to the existing convention in CSA to have the `context` parameter first.
 
+#### js-implicit
+
+If you want JavaScript linking for a builtin defined in Torque, you should use keyword `js-implicit` instead of `implicit`. The arguments are limited to these four components of the calling convention:
```

Comment:
This sounds a bit like js-implicit causes JavaScript linkage. Perhaps we should say something like:
"For builtins with JavaScript linkage defined in Torque, you ..."

Reply:
Good point. done.